frontend/src/component/ui/ViewChangeLogButton.spec.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { render, fireEvent } from '@testing-library/react';
+import { ViewChangeLogButton } from './ViewChangeLogButton';
+
+it('renders without crashing and handles click events correctly', () => {
```

Comment:
Let's separate those two test cases when one of them fails, we know exactly what causes the failure.